.gitignore

```diff
@@ -1,2 +1,3 @@
 .idea/workspace.xml
 __pycache__/
+map_converter.py
```

Comment:
Why is map_converter.py in the gitignore? That would prevent GitHub from tracking changes to the file....

Reply:
Because it's not a necessary or used in the code, it was solely for testing purposes. All the code written in it was integrated into the other files after I had tested it.

Reply:
I agree with @amistele ... the code does not use it or need it.